Description: fix install path for example SVG files
Origin: Fedora
Author: Helio Chissini de Castro <helio@kde.org>
Last-Update: 2016-02-20

--- a/examples/svg/richtext/textobject/textobject.pro
+++ b/examples/svg/richtext/textobject/textobject.pro
@@ -13,5 +13,5 @@
 INSTALLS += target
 
 filesToDeploy.files = files/*.svg
-filesToDeploy.path = files
+filesToDeploy.path = $$[QT_INSTALL_EXAMPLES]/svg/richtext/textobject/files
 DEPLOYMENT += filesToDeploy
